pages/index.tsx

```diff
@@ -62,7 +54,35 @@ const Home: NextPage<ExperiencePageProps> = ({experience}) => {
                             p: '10px'
                             }}
                         >
-                            <LoadingButtonIndicator loading={true} loadingPosition={'start'} variant={'contained'} height={'60px'} width={'200px'} loadingInput={'Loading Quiz'} />
+
+                    <LoadingIndicator 
+                        loadingInput={'Loading Quiz'}
+                        loading={true} 
+                        loadingPosition={'start'}
+                        varient={'text'}
+                        sx={{
+                                height:'60px', 
+                                width:'175px'}}>
+                        
+                        <Link href="/curious" underline="none"
+                            sx={{
+                                textTransform: 'uppercase',
+                                fontWeight: '500',
+                                color: 'white',
+                                backgroundColor: '#2ca2a3',
+                                borderRadius: '4px',
+                                boxShadow: '0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12)',
```

Comment:
added a lot of formatting to Link so that it would look like all the other buttons throughout the page. is this cheating haha; im sure there is probably a better way of doing this.

Reply:
+ should I edit the "Retake Quiz" link on QuizResults to look like a button as well?